mne/epochs.py

```diff
@@ -1198,8 +1198,15 @@ def plot_image(self, picks=None, sigma=0., vmin=None,
             The scalings of the channel types to be applied for plotting.
             If None, defaults to `scalings=dict(eeg=1e6, grad=1e13, mag=1e15,
             eog=1e6)`.
-        cmap : matplotlib colormap
-            Colormap.
+        cmap : matplotlib colormap | (colormap, bool) | 'interactive'
+            Colormap. If tuple, the first value indicates the colormap to use
+            and the second value is a boolean defining interactivity. In
+            interactive mode the colors are adjustable by clicking and dragging
+            the colorbar with left and right mouse button. Left mouse button
+            moves the scale up and down and right mouse button adjusts the
+            range. Hitting space bar resets the scale. Up and down arrows can
+            be used to change the colormap. If 'interactive', translates to
+            ('RdBu_r', True). Defaults to 'RdBu_r'.
```

Comment:
AFAIK we don't use RdBu_r all the time eg if we plot grads norm. Write

If 'interactive', the default colormap is used with interactive support.